### DISCUSSION TITLE
Audio Passthrough

### DISCUSSION THREAD

**Discussion by pi43r** [notAperson535/OneClick-macOS-Simple-KVM#23] [Q&A] [answered]
Hi,

Thanks for the package, it was the only one that worked for me so far.
I was able to install Monterey on an Intel 8550U with Fedora36 and works pretty stable, except some visual lags sometimes.

Now I'm trying to get audio working and I was wondering if it's possible to pass audio through to the host machine. I guess I could also do PCI or USB (external sound interface) passthrough, but then I would lose the audio connection from my host.

Any way to connect audio easily? (using the shell script, because virt-manager is not working for me on Fedora)

I'm getting this error message when starting the VM, but this seems to be expected:
```
qxl_send_events: spice-server bug: guest stopped, ignoring
sdl: SDL_OpenAudioDevice for playback failed
sdl: Reason: ALSA: Couldn't open audio device: Host is down
sdl: SDL_OpenAudioDevice for playback failed
sdl: Reason: ALSA: Couldn't open audio device: Host is down
audio: Failed to create voice `dac'
sdl: SDL_OpenAudioDevice for recording failed
sdl: Reason: ALSA: Couldn't open audio device: Host is down
sdl: SDL_OpenAudioDevice for recording failed
sdl: Reason: ALSA: Couldn't open audio device: Host is down
audio: Failed to create voice `adc'
```

**Comment by notAperson535** [comment, marked as answer]
@pi43r [Here](https://github.com/kholia/OSX-KVM/blob/master/notes.md#virtual-sound-device) it says you can get a cheap audio card and use it, but I do not know many details about that. I think it is saying you could get a cheap audio card and pass it through.
